src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -91,6 +91,9 @@
     @FindBy(id = "button_submit_add")
     private WebElement addNewQuestionButton;
     
+    @FindBy(id = "button_done_editing")
+    private WebElement doneEditingLink;
```

Comment:
Although this was named "Link" earlier, this is actually a "Button", so it's better if you name it that way.